test/run_test.py

```diff
@@ -834,11 +834,11 @@ def parse_args():
         help='exclude tests that are run for a specific jit config'
     )
     parser.add_argument(
-        '--run-specified-test-cases',
-        nargs='?',
+        '--use-specified-test-cases',
         type=str,
-        const=SPECIFIED_TEST_CASES_FILE,
-        help='runs specified test cases from previous OSS CI stats from a file, format CSV',
+        choices=['include', 'bring-to-front'],
+        help='if this flag is set. run_test will load specified test cases file from '
+        'SPECIFIED_TEST_CASES_FILE and use the list to determine what test should be run',
```

Comment:
So we'd just force users to always name their file to SPECIFIED_TEST_CASES_FILE? Is there a way to do this where we still allow the user to specify their own custom file name?

Reply:
i don't think `run_test.py` is for user directly - since developer should directly use `test/test_*.py` which calls common_utils.run_test() or other customized handlers. Do you think we have use case when user will do this?